README.md

```diff
@@ -0,0 +1,21 @@
+# Joke Farm
+
+## Install dependencies
+Install the dependencies with: `npm install `
+
+## Run
+    `npm run dev`
```

Comment:
In Markdown, any indented block of text is interpreted as a `<pre>` block.
So these two lines should have been :

```

    npm run dev

view it in <http://localhost:8080>
```